pkg/base/helm_v3.go

```diff
@@ -84,13 +77,24 @@ func renderHelmV3(chartName string, chartPath string, vals map[string]interface{
 		},
 	}
 
+	var manifests bytes.Buffer
+	fmt.Fprintln(&manifests, strings.TrimSpace(rel.Manifest))
+	// add hooks
+	for _, m := range rel.Hooks {
+		fmt.Fprintf(&manifests, "---\n# Source: %s\n%s\n", m.Path, m.Manifest)
+	}
+	// add crds
+	for _, crd := range chartRequested.CRDObjects() {
+		fmt.Fprintf(&manifests, "---\n# Source: %s\n%s\n", crd.Filename, string(crd.File.Data[:]))
+	}
+
 	splitManifests := splitManifests(manifests.String())
 	manifestName := ""
 	for _, manifest := range splitManifests {
 		submatch := HelmV3ManifestNameRegex.FindStringSubmatch(manifest)
 		if len(submatch) > 0 {
 			// multi-doc manifests will not have the Source comment so use the previous name
-			manifestName = strings.TrimPrefix(submatch[1], fmt.Sprintf("%s/", chartName))
+			manifestName = strings.TrimPrefix(submatch[1], fmt.Sprintf("%s/", chartRequested.Name()))
```

Comment:
Fixes a separate issue where if `spec.chart.releaseName` was specified, the chart name prefix was not removed here since it would search for the release name instead.  This changes it to always search for the chart name.